pyproject.toml

```diff
@@ -7,7 +7,7 @@ build-backend = "setuptools.build_meta"
 
 [tool.black]
 line-length = 88
-target-version = ['py37', 'py38', 'py39', 'py310']
+target-version = ['py39', 'py310']
```

Comment:
@MattToast @ashao @AlyssaCote Is there a reason we don't have py311 here?

Reply:
I'd vote to sneak it in on this PR since it shouldn't hurt anything

Reply:
i'm on it!